dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,52 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Attempt to enroll the key in the incorrect format (OS)
+    [Documentation]    This test verifies that it is impossible to load
```

Comment:
@WiktorG351 please lets use more descriptive about what this test does; creation of key and certs with openssl, using sbctl enroll them

Reply:
@TomaszAIR adressed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/150/commits/5811b042862c128816abe8ab4ab2a575d7eedb5b#diff-a85ad2551c590f23fcfdd0995aeb59134f1c5600bec9d6b563798c00053e51d4R238-R239)